tests/PathBrokerTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Northwoods\Broker;
+
+use League\Container\Container;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Zend\Diactoros\Response\HtmlResponse;
+use Zend\Diactoros\ServerRequest;
+
+class PathBrokerTest extends TestCase
+{
+    use CanMock;
```

Comment:
I see no reference to the implementation of this trait.

Reply:
I am calling `$this->process()`.

Reply:
Oh you're reusing an existing trait. Been a while since I worked on this. 😉